res/css/views/elements/_InlineSpinner.scss

```diff
@@ -18,7 +18,24 @@ limitations under the License.
     display: inline;
 }
 
-.mx_InlineSpinner_spin img {
+.mx_InlineSpinner img, .mx_InlineSpinner_icon {
     margin: 0px 6px;
     vertical-align: -3px;
 }
+
+@keyframes spin {
+    from {
+        transform: rotateZ(0deg);
+    }
+    to {
+        transform: rotateZ(360deg);
+    }
+}
+
+.mx_InlineSpinner_icon {
+    display: inline-block;
+    background-color: $primary-fg-color;
+    mask: url('$(res)/img/spinner.svg');
+    mask-size: contain;
+    animation: 1.1s steps(12, end) infinite spin;
+}
```

Comment:
I'd be okay with us just using the `mx_Spinner_icon` class instead of copy/pasting it for inline spinners.

Reply:
:heavy_check_mark: Done!